spec/unit/mutant/result/subject_spec.rb

```diff
@@ -106,4 +106,30 @@
       end
     end
   end
+
+  describe '#neutral_failure?' do
+    subject { object.neutral_failure? }
+
+    context 'without failing neutral mutations' do
+      let(:mutation_results) do
+        [
+          instance_double(Mutant::Result::Mutation, success?: false, neutral_failure?: false),
+          instance_double(Mutant::Result::Mutation, success?: true, neutral_failure?: false)
```

Comment:
I'd probably vertically align the `neutral_failure?` key here for easier readability.